MonkeyBot/Documentation/DocumentationBuilder.cs

```diff
@@ -128,7 +128,7 @@ private static string TranslatePrecondition(PreconditionAttribute precondition,
             }
             else
             {
-                return precondition.ToString();
+                return precondition.ErrorMessage;
```

Comment:
why this change?

Reply:
I wanted the methods return type to be non-nullable (otherwise I would introduce a cascade of new warnings), but `ToString()` for some reason returns nullable string, so I wanted to use something similar and noticed that the error message wasn't nullable. I was definitely a bit uncertain of the correctness of this fix - especially not having a setup to run the bot myself. :-)

Reply:
ErrorMessage is only set when the Precondition is actually checked before running a command if I interpret it correctly. It defaults to null.
But just leave as is, I just looked at the whole method and it just asks for a refactoring to a switch expression anyway :)